test/cli/cli_test.go

```diff
@@ -0,0 +1,191 @@
+package cli
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"runtime"
+	"strconv"
+	"strings"
+	"testing"
+
+	"github.com/google/shlex"
+	"github.com/joho/godotenv"
+	"github.com/otiai10/copy"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/keboola/processor-split-table/internal/pkg/utils"
+	"github.com/keboola/processor-split-table/test"
+)
+
+// TestCLI runs all data-dir tests from the file directory.
+// In "envs" and "args" fi $IN_DIR and $OUT_DIR placeholder.
+func TestCLI(t *testing.T) {
+	t.Parallel()
+
+	_, testFile, _, _ := runtime.Caller(0)
+	rootDir := filepath.Dir(testFile)
+
```

Comment:
E2E testy pre CLI, vid dalej, podobne ako pre processor, ale nie rovnake.
Kedze sa CLI/processor testy prekryvaju, tak pridam iba par CLI testov, na vyskusanie, ze to ide E2E.